x/blob/client/cli/payforblob.go

```diff
@@ -57,10 +69,19 @@ func CmdPayForBlob() *cobra.Command {
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
+	cmd.PersistentFlags().Uint8(FlagShareVersion, 0, "Specify the share version")
 
 	return cmd
 }
 
+func validateShareVersion(shareVersion uint8) error {
+	// Check if share version is present in SupportedShareVersions
+	if !slices.Contains(appconsts.SupportedShareVersions, shareVersion) {
+		return fmt.Errorf("share version %d is not supported", shareVersion)
+	}
+	return nil
+}
```

Comment:
after simplifying this, we're performing these exact checks when we validate the blobs during the creation of the PFB, so we can actually remove this function entirely

https://github.com/celestiaorg/celestia-app/blob/ab24a7b4d5190a53cf4bb0a9e989f4c824759f8e/x/blob/types/payforblob.go#L29-L30